test_crates/trait_associated_type_default_removed/new/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+mod sealed {
```

Comment:
Associated type defaults are unstable, but can be used with a nightly compiler and by explicitly enabling the feature in the crate's `lib.rs`:
```suggestion
#![feature(associated_type_defaults)]

mod sealed {
```